docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/net/index.md

```diff
@@ -7,7 +7,85 @@ description: The simplest way to start capturing telemetry data is to implement
 
 Automatic instrumentation of the .NET applications is a very easy task. The simplest way to start capturing telemetry data is to implement the solution coming from OpenTelemetry-dotNet. All the libraries shipped with the [OpenTelemetry-dotNet](https://github.com/open-telemetry/opentelemetry-dotnet) repository support all the officially supported versions of .NET Core (including deployments in the [Microsoft Azure Service Fabric Containers](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-containers-overview)) and .NET framework with an except for .NET Framework 3.5 SP1. See this [list of the supported libraries](https://github.com/open-telemetry/opentelemetry-dotnet#getting-started).
 
-## How to instrument your ASP.NET Core application
+# .Net full auto-instrumentation vs library-instrumentation (partial auto)
+
+It is important to understand difference between two types of instrumentations that
+are available in .Net. First one, is fully automatic one that does not require
+code. Second one, requires some initialization that has to be done in the code
+and is called library instrumentation by opentelemetry community. It is partial
+auto-instrumentation as traces are generated automatically depending on settings
+provided by developer(s) during initialization phase.
+
+## How to instrument .NET application, fully automaticly (Windows)
```

Comment:
Please mention about (for both `.net` and `iis`)
- exact version of the auto-instrumentation used in this guide
- configuration possibilities (e.g. for advanced usage add link to the official documentation matching version)
- configuration related to traces export (env vars required to export traces, please use http/protobuf as default), extra resource attributes, service name